lib/contentful.js

```diff
@@ -246,12 +246,11 @@ module.exports.renderAllMessagesForPhoenixCampaign = function (phoenixCampaign)
   return new Promise((resolve, reject) =>
     // get all fields, overrides and default
     exports.getAllFieldsForCampaign(phoenixCampaign.id)
-      .then((campaignFields) => { // eslint-disable-line arrow-body-style
-        /**
-         * Use Promise.map to create an array of promises to wait on,
-         * replacing the push->forEach->Promise.all boilerplate
-         */
-        return Promise.map(
+      /**
+       * Use Promise.map to create an array of promises to wait on,
+       * replacing the push->forEach->Promise.all boilerplate
```

Comment:
Does this comment make sense on its own (outside of the pull request where we could see the `push` / `Promise.all` it's replacing)?